core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -71,6 +69,8 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
   private var zkUtils: ZkUtils = null
   private var aclChangeListener: ZkNodeChangeNotificationListener = null
 
+  private val NoVersion = -2
```

Comment:
I should clean, that up. I just needed some way to indicated there is no version yet, and -1 in the Zookeeper api means "ignore the version". I can rework the code to eliminate this, or use Option/None.